legate/core/runtime.py

```diff
@@ -1038,6 +1138,16 @@ def num_omps(self) -> int:
     def num_gpus(self) -> int:
         return self._num_gpus
 
+    @property
+    def num_nodes(self) -> int:
+        return self._num_nodes
+
+    def machine_model(self) -> MachineModel:
+        return self._machine_model
+
+    def get_next_available_point(self) -> DeviceID:
```

Comment:
maybe we shouldn't add this shortcut but let the caller go through the machine model instead.